### PR TITLE
use laminas/laminas-diactoros insted of zendframework/zend-diactoros

### PR DESCRIPTION
**Pull request by andrii-pukhalevych**
https://github.com/cakephp/authentication/issues/385

### CODE REVIEW

**Comment by othercorey** [comment]
You need to merge master branch to resolve conflicts.

File: src/Middleware/AuthenticationMiddleware.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 use RuntimeException;
-use Zend\Diactoros\Stream;
+use Laminas\Diactoros\Stream;
```
Comment (othercorey) [inline src/Middleware/AuthenticationMiddleware.php:26]: This needs to be in alphabetical order.

**Comment by andrii-pukhalevych** [comment]
> You need to merge master branch to resolve conflicts.

@othercorey How I can do this?

**Comment by othercorey** [comment]
You need to pull upstream into you fork and merge upstream master into your branch.

https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

**Pull request closed without merging**